### PR TITLE
Chromatic stories consolidation

### PR DESCRIPTION
**Pull request by larseirikhansen**
### Description

https://github.com/orgs/navikt/projects/49/views/18?pane=issue&itemId=52366151

To avoid a huge Chromatic bill we attempt to reduce how many times chromatic runs on every Storybook component by creating a Chromatic-component in each Storybook-file that consolidates the components there.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -163,7 +163,7 @@
     "@typescript-eslint/eslint-plugin": "^6.7.3",
     "@typescript-eslint/parser": "^6.7.3",
     "@whitespace/storybook-addon-html": "^6.1.1",
-    "chromatic": "11.3.0",
+    "chromatic": "11.5.3",
```
Comment (JulianNymark) [inline package.json]: also decided to try update chromatic as part of the attempt to fix strange behaviour with `forcedColors`, but it didn't seem to work, decided to leave the update in the PR though (it's minor, so shouldn't break anything & require too much testing).

File: @navikt/core/react/src/button/button.stories.tsx [hunk 2]
```diff
@@ -187,6 +187,6 @@ export const Chromatic: Story = {
 export const ChromaticForcedColors: Story = {
   render: Chromatic.render,
   parameters: {
-    chromatic: { disable: false, forcedColors: "active" },
+    chromatic: { disable: false },
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/button/button.stories.tsx]: Please don't merge this change (at least not yet). I have contacted Chromatic about this bug, so need it to be reproducible 🙂

Comment (HalvorHaugan) [inline @navikt/core/react/src/button/button.stories.tsx]:
```suggestion
    chromatic: { disable: false, forcedColors: "active" },
```
They say they have fixed it, so trying to revert.

Comment (HalvorHaugan) [reply @navikt/core/react/src/button/button.stories.tsx]: Seems to be fixed now 🙌

Comment (JulianNymark) [reply @navikt/core/react/src/button/button.stories.tsx]: nice! (Chromatic was amazingly fast! felt like they did this fix just for us! 🙌 )

Comment (HalvorHaugan) [reply @navikt/core/react/src/button/button.stories.tsx]: Yeah, probably not many people using forcedColors 😅

File: @navikt/core/react/src/layout/bleed/Bleed.stories.tsx [hunk 3]
```diff
@@ -393,3 +396,22 @@ export const ReflectivePadding = {
...
+      <h2>Breakpoints</h2>
+      <Breakpoints.render />
+      <h2>Px</h2>
+      <Px.render />
+      <h2>Full</h2>
+      <Full.render />
+      <h2>ReflectivePadding</h2>
+      <ReflectivePadding.render />
+    </VStack>
+  ),
```
Comment (KenAJoh) [inline @navikt/core/react/src/layout/bleed/Bleed.stories.tsx:416]:
```suggestion
  ),
  parameters: {
    chromatic: { disable: false },
  },
```

File: @navikt/core/react/src/layout/responsive/show.stories.tsx [hunk 4]
```diff
@@ -78,3 +85,20 @@ export const AsChild = {
...
+    <VStack gap="4">
+      <h2>Default</h2>
+      {Default.render?.(props, context)}
+      <h2>AsChild</h2>
+      {AsChild.render?.(props, context)}
+    </VStack>
+  ),
+};
+
+Chromatic.parameters = {
```
Comment (KenAJoh) [inline @navikt/core/react/src/layout/responsive/show.stories.tsx]: Can be moved up into `Chromatic`-story

**Review by KenAJoh — APPROVED** [review]
LGTM 🚀 Tenker jeg bare merger denne nå før sommerferien, så kan det evt bli en ny PR hvis det gjenstår noe 🙌

**Pull request merged**